v22.2/node-shutdown.md

```diff
@@ -800,6 +796,62 @@ If the decommissioning node has already reached the [draining stage](#draining),
 On the **Cluster Overview** page of the DB Console, the [node status](ui-cluster-overview-page.html#node-status) of the node should be `LIVE`. After a few minutes, you should see replicas rebalanced to the nodes.
 </section>
 
+## Decommissioning and draining on Kubernetes
+
+Most of the guidance in this page is most relevant to manual deployments that don't use Kubernetes. If you use Kubernetes to deploy CockroachDB, draining and decommissioning work the same way for the `cockroach` process, but Kubernetes handles them on your behalf. In a Kubernetes deployment, you don't usually decommission or drain a node interactively. In fact, if you attempt to do so, Kubernetes will automatically undo your change so that the cluster remains in its desired state as defined by the Kubernetes deployment. Instead, each cluster node is managed within Kubernetes.
```

Comment:
Avoid contractions where a clear instruction is given - see style guide on this.

```suggestion
Most of the guidance in this page is most relevant to manual deployments that don't use Kubernetes. If you use Kubernetes to deploy CockroachDB, draining and decommissioning work the same way for the `cockroach` process, but Kubernetes handles them on your behalf. In a Kubernetes deployment, you do not usually decommission or drain a node interactively. In fact, if you attempt to do so, Kubernetes will automatically undo your change so that the cluster remains in its desired state as defined by the Kubernetes deployment. Instead, each cluster node is managed within Kubernetes.
```